build.gradle

```diff
@@ -1,22 +1,18 @@
 plugins {
-  id 'org.owasp.dependencycheck' version '8.0.2'
+  id 'org.owasp.dependencycheck' version '7.3.0'
   id 'scala'
-  id 'com.github.lkishalmi.gatling' version '3.0.2'
-  //id 'java-gradle-plugin'
-  id 'groovy'
-  id 'com.gradle.plugin-publish' version '0.9.8'
-  id 'maven-publish'
+  id 'java-library'
+  id 'io.gatling.gradle' version '3.8.4'
```

Comment:
Please follow the versions in the table at the top of https://tools.hmcts.net/confluence/display/RQA/Gatling+-+Upgrading+Versions+and+Dependencies as you're using old version of gatling and scala

Reply:
This has been changed